go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/hashicorp/terraform-provider-consul
 
 require (
-	github.com/hashicorp/consul/api v1.14.0
+	github.com/hashicorp/consul/api v1.16.0
```

Comment:
I think API version release with 1.14.2 is 1.18.0 https://github.com/hashicorp/consul/blob/0ba7a4018304e81c370b662aa435fee45d287076/go.mod#L34

Reply:
Thanks.